src/lib/daemon_rpcs/daemon_rpcs.ml

```diff
@@ -177,15 +177,17 @@ module Types = struct
 
       let blockchain_length = int_option_entry "Block Height"
 
-      let uptime_secs = map_entry "Local Uptime" ~f:(sprintf "%ds")
+      let uptime_secs =
+        map_entry "Local Uptime" ~f:(fun secs ->
+            Time.Span.to_string (Time.Span.of_int_sec secs) )
 
       let ledger_merkle_root = string_option_entry "Ledger Merkle Root"
 
       let staged_ledger_hash = string_option_entry "Staged-ledger Hash"
 
       let state_hash = string_option_entry "Staged Hash"
 
-      let commit_id = string_entry "GIT SHA1"
+      let commit_id = string_entry "GIT SHA-1"
 
       let conf_dir = string_entry "Configuration Directory"
 
```

Comment:
We should use another term instead of user_commands - would `Transactions sent` suffice? Or does it also include fee transfers? I think of both of these as txns...

in the context of a CLI, I can see how user_commands might be mistaken to be client commands sent

Reply:
User commands includes changing your delegation.

Reply:
User command: payment + delegations
Txns: usercommand + fees + coinbase
more or less

This count is the number of transactions sent by this client/daemon as far as I can tell.